packages/app/package.json

```diff
@@ -28,7 +28,7 @@
     "@intlify/vite-plugin-vue-i18n": "2.4.0",
     "@packages/frontend-shared": "0.0.0-development",
     "@percy/cypress": "^3.1.0",
-    "@testing-library/cypress": "8.0.0",
+    "@testing-library/cypress": "BlueWinds/cypress-testing-library#119054b5963b0d2e064b13c5cc6fc9db32c8b7b5",
```

Comment:
This is a fork of cypress-testing-library which uses queries.

My current plan is to wait for this PR to merge into issue-7306, which will build the cypress binary; I can then update my fork of `cypress-testing-library` to point to that branch for its own tests, and open a PR upstream.